specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -4762,7 +4762,7 @@
           "additionalProperties": {
             "type": "string"
           },
-          "maxItems": 1000
+          "maxItems": 50
```

Comment:
Do we have validation on key/value lengths as well, like ARM does? We need to make sure that we scale properly when someone creates the largest possible invoice section entities, many times.

Reply:
Yes we have validation on key/value lengths as well.

Reply:
We should describe the validation in the description of this property so people can look this up

Reply:
added